code/check.py

```diff
@@ -1,32 +1,36 @@
+#!/usr/bin/env python
 import sys
 import numpy
 
 
 def main():
-    script = sys.argv[0]
     filenames = sys.argv[1:]
-    if len(filenames) <= 1:  # nothing to check
-        print('Only 1 file specified on input')
+    total_files = len(filenames)
+    if total_files <= 1:  # nothing to check
+        print(f'''{'Only one file' if total_files else 'None files'} specified
+        on input''')
```

Comment:
I think this syntax could be confusing to the novice learners we are targeting with this lesson. I don't think we cover it exactly.

Reply:
Fair point, I should remove this, it's a shame since loop comprehension is so nice.